helm/portieris/templates/default/policies.yaml

```diff
@@ -51,22 +54,21 @@ spec:
       policy:
     - name: "registry*.bluemix.net/armada-master/*"
       policy:
+    {{ end }}
 ---
 apiVersion: securityenforcement.admission.cloud.ibm.com/v1beta1
 kind: ClusterImagePolicy
 metadata:
-  name: ibmcloud-default-cluster-image-policy
+  name: default-cluster-image-policy
```

Comment:
Should this have Portieris in the name somewhere to make it more likely to be unique in the cluster?